core/shared/src/main/scala/zio/ZIO.scala

```diff
@@ -176,11 +182,13 @@ sealed trait ZIO[-R, +E, +A] extends Serializable with ZIOPlatformSpecific[R, E,
    * val parsed = readFile("foo.txt") >>= parseFile
    * }}}
    */
+  @deprecated("use flatMap", "2.0.0")
   final def >>=[R1 <: R, E1 >: E, B](k: A => ZIO[R1, E1, B]): ZIO[R1, E1, B] = flatMap(k)
```

Comment:
👍 for removing this. This one is a little unusual in that it combines values of two different types unlike other binary operators. It is also something that you actually see used sometime which can make code bases less accessible.